samples/system-test/spanner.test.js

```diff
@@ -489,6 +490,18 @@ describe('Spanner', () => {
     );
   });
 
+  // query with RPC priority
+  // TODO: Enable when RPC Priority has been released.
+  it.skip('should use request options', async () => {
```

Comment:
If we choose to keep the sample then you can unskip this now.

Reply:
Argh. I missed this one and went ahead and merged without it. I've opened a new PR for enabling it here: https://github.com/googleapis/nodejs-spanner/pull/1338